bundler/spec/bundler/stub_specification_spec.rb

```diff
@@ -55,4 +55,24 @@
       expect(stub.missing_extensions?).to be true
     end
   end
+
+  describe "#activated?" do
+    it "returns true after activation" do
+      stub = described_class.from_stub(with_bundler_stub_spec)
+
+      expect(stub.activated?).to be false
+      stub.activated = true
+      expect(stub.activated?).to be true
+    end
+
+    it "returns true after activation if the underlying stub is a `Gem::StubSpecification`" do
+      spec_path = File.join(File.dirname(__FILE__), "specifications", "foo.gemspec")
+      gem_stub = Gem::StubSpecification.new(spec_path, File.dirname(__FILE__),"","")
+      stub = described_class.from_stub(gem_stub)
+
+      expect(stub.activated?).to be nil
```

Comment:
While the other test returns `false` here, `nil` is falsy and I think that's fine.

Reply:
Yeah, I get we could make it consistent by making sure `Gem::StubSpecification#activated?` returns only true/false, but I honestly think it's fine as is. We could use `be_falsey` though to make these two tests consistent.

Reply:
Nice suggestion, I changed it to `be_falsy`.